README.md

```diff
@@ -15,6 +15,7 @@ Pacman for Emacs
 ## Requirements ##
 
 - Emacs 24.3+
+- [Cask][cask] 0.7.4+
```

Comment:
I was planning this section as the Requirements for the end user. And Cask is more a development requirement. Maybe we can create another section for the development requirements or simply remove this item from here 'cause you already mentioned Cask in the Development section.